src/gui/screen_menu_esp_update.cpp

```diff
@@ -203,7 +203,7 @@ void ScreenMenuESPUpdate::windowEvent(EventLock /*has private ctor*/, window_t *
         case esp_upload_action::Start_flash:
             if ((opened_file = fopen(current_file->filename, "rb")) == nullptr) {
                 log_error(Network, "ESP flash: Unable to open file %s", current_file->filename);
-                help.SetText(_("Unable to open files. Chek the flash drive and start again"));
+                help.SetText(_("Unable to open files. Try to re-flash the last FW and try again"));
```

Comment:
Reflashing the firmware won't help. I don't think there's currently any user-accessible way to try fix that if that problem persists.

However, trying to flash the ESP again sometimes helps.

Reply:
Well, I have seen it to fail somewhat randomly. It's probably some race condition or noise on some internal bus. Or it may be related to me powering the knoblet off a USB. Retrying did help.

Either way, if it fails, it is a bug ‒ either because it allowed it to load wrong file to xflash, or because the above problem. Retrying is a workaround that might help ‒ but it's the only one we can reasonably give. Because reflashing the firmware will _not trigger_ reloading of the files to xflash.